requirements.txt

```diff
@@ -29,7 +29,6 @@ pytest-xdist==2.2.0
 pytorch_transformers
 pyyaml
 rich
```

Comment:
Are you sure we're still using this? I'm fairly certain we don't have rich terminal anymore.

Reply:
Same for prettytable, also not being used anymore.